frontend/src/context/search-org/types.ts

```diff
@@ -0,0 +1,4 @@
+export const searchQueryKeys = ["name"];
```

Comment:
It seems a little odd to be using an array of objects with just a single key here, for collections I think we could just pass an array of strings into Fuse, and omit the query key. That said, if we want to extend this to encompass archived items/workflows in the future, and/or we want to add descriptions to the search values for collections, I could see this being extended, in which case it'd be nice to have the values from the Fuse collections typed better by search org key so that we could do `item.name` instead of `item["name"]`, etc. None of this is requesting a change right now necessarily, just sharing thoughts for the future! Might be nice to add some comments making note of this though.